src/coreclr/jit/hwintrinsic.h

```diff
@@ -606,6 +606,27 @@ struct HWIntrinsicInfo
         return lookup(id).flags;
     }
 
+    static bool IsXCNTIntrinsic(NamedIntrinsic id)
+    {
+        switch (id)
+        {
+            case NI_System_Numerics_BitOperations_PopCount:
```

Comment:
This is missing `ArmBase.LeadingZeroCount` and `ArmBase.LeadingSignCount`

Reply:
It also should not specify `NI_System_Numerics_BitOperations_PopCount`, that is not actually an intrinsic in IR (should be dissolved into one of the platform-specific ones at this point).

Reply:
Should we include ` VectorX`?

Reply:
> Should we include ` VectorX.get_Count`?

no need, those should be imported as constants early in the importer

Reply:
Updated in eea3f3aab99fed049beb4c674eb95d9b92bdda58